pyodide_lock/cli/lockfile.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+
+try:
+    import typer
```

Comment:
would move this to `cli.py`

Reply:
oh, it already is... if this needs `typer`, that's fine, but the huge try check was confusing.